internal/cmd/options.go

```diff
@@ -3,7 +3,9 @@ package cmd
 import (
 	"context"
 
+	"github.com/notaryproject/notation-go/log"
 	"github.com/notaryproject/notation/internal/trace"
+	executabletrace "github.com/oras-project/oras-credentials-go/trace"
```

Comment:
I think the name `executabletrace` looks like it traces any kind of executables, while it just traces executables for credentials usage.

Reply:
changed the package name to `credentialstrace`.